app/controllers/works_controller.rb

```diff
@@ -26,7 +26,7 @@ def create
 
     @form = work_form(work_version)
     if @form.validate(work_params) && @form.save
-      after_save(form: @form, event_context: { user: current_user, description: 'Created' })
+      after_save(form: @form, event_context: build_event_context(form: @form, event_type: :create))
     else
```

Comment:
We want to record changes for a new work, but the logic is bit different.  That logic will be handled in the class, but we need to tell it this is a new work.